cms/templates/product_page.html

```diff
@@ -67,16 +67,32 @@ <h1>{{ page.title }}</h1>
                   </section>
                 </div>
 
-              <nav id="tab-bar">
+            <div data-bs-spy="scroll" data-bs-target="#tab-bar" tabindex="0" data-bs-offset=100 class="scroll-spy-content">
+              {# Non-Mobile only #}
+              <nav id="tab-bar" class="sticky-top d-none d-md-block">
                 <ul class="nav">
-                  {% if page.about %}<li class="nav-item"><a href="#about-this-class">About</a></li>{% endif %}
-                  {% if page.is_program_page %}<li class="nav-item"><a href="#program-courses">Courses</a></li>{% endif %}
-                  {% if page.what_you_learn %}<li class="nav-item"><a href="#what-youll-learn">What you'll learn</a></li>{% endif %}
-                  {% if page.prerequisites %}<li class="nav-item"><a href="#prerequisites">Prerequisites</a></li>{% endif %}
-                  {% if instructors %}<li class="nav-item"><a href="#instructors">Instructors</a></li>{% endif %}
-                  {% if page.faq_url %}<li class="nav-item"><a href="{{page.faq_url}}" target="_blank">FAQs</a></li>{% endif %}
+                  {% if page.about %}<a href="#about-this-class"><li class="nav-item">About</li></a>{% endif %}
```

Comment:
I believe that `li` is the only valid direct child of `ul` and `ol` elements—this nesting is invalid according to HTML5. See https://html.spec.whatwg.org/#the-ul-element with
```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <title>Hello, world</title> 
  </head>
  <body>
    <ul>
      <a><li>Item 1</li></a>
      <a><li>Item 1</li></a>
    </ul>
  </body>
</html>
```

Browsers probably tolerate this just fine, but IMO it would be better to use `li` as the direct descendant. 

What was the reason to swap the `li` and `a` order? Maybe could achieve the goal some other way.